### ISSUE TITLE
Impossible to compile Android/ios

### ISSUE THREAD

**Issue by Araknos** [NativeScript/nativescript-schematics#193]
Hey, I got an angular web app and I want to transform it into mobile app with nativescript. I just followed this https://www.youtube.com/watch?v=WdhJZ8kGXvw and ran:
`ng add @nativescript/schematics`

But got this error when I ran `tns run ios --bundle`
```
Applications for platform iOS can not be built on this OS
# tns run ios

### Description

Runs your project on a connected iOS device or in the iOS Simulator, if configured. This is shorthand for prepare, build and deploy. While your app is running, prints the output from the application in the console and watches for changes in your code. Once a change is detected, it synchronizes the change with all selected devices and restarts/refreshes the application.

WARNING: You can run this command only on macOS systems. To view the complete help for this command, run $ tns help run ios
```

Not working too with `tns run android --bundle`

```
√ Local builds for iOS can be executed only on a macOS system. To build for iOS on a different operating system, you can use the NativeScript cloud infrastructure.
× The ANDROID_HOME environment variable is not set or it points to a non-existent directory. You will not be able to perform any build-related operations for Android.
 To be able to perform Android build-related operations, set the `ANDROID_HOME` variable to point to the root of your Android SDK installation directory.

× WARNING: adb from the Android SDK is not installed or is not configured properly.
 For Android-related operations, the NativeScript CLI will use a built-in version of adb.
To avoid possible issues with the native Android emulator, Genymotion or connected
Android devices, verify that you have installed the latest Android SDK and
its dependencies as described in http://developer.android.com/sdk/index.html#Requirements

× WARNING: The Android SDK is not installed or is not configured properly.
 You will not be able to run your apps in the native emulator. To be able to run apps
in the native Android emulator, verify that you have installed the latest Android SDK
and its dependencies as described in http://developer.android.com/sdk/index.html#Requirements

× Cannot find a compatible Android SDK for compilation. To be able to build for Android, install Android SDK 28 or later.
 Run `$ sdkmanager` to manage your Android SDK versions.

× You need to have the Android SDK Build-tools installed on your system. You can install any version in the following range: '>=23 <=28'.
 Run `$ sdkmanager` from your command-line to install required `Android Build Tools`. In case you already have them installed, make sure `ANDROID_HOME` environment variable is set correctly.

× Error executing command 'javac'. Make sure you have installed The Java Development Kit (JDK) and set JAVA_HOME environment variable.
 You will not be able to build your projects for Android.
To be able to build for Android, verify that you have installed The Java Development Kit (JDK) and configured it according to system requirements as
 described in http://docs.nativescript.org/setup/ns-cli-setup/ns-setup-win.html#system-requirements

× WARNING: The Java Development Kit (JDK) is not installed or is not configured properly.
 You will not be able to work with the Android SDK and you might not be able
to perform some Android-related operations. To ensure that you can develop and
test your apps for Android, verify that you have installed the JDK as
described in http://docs.oracle.com/javase/8/docs/technotes/guides/install/install_overview.html (for JDK 8).
```
I guess I forgot something

**Comment by Swoox** [comment]
Try `tns doctor`
[](https://docs.nativescript.org/tooling/docs-cli/general/doctor)
